### DISCUSSION TITLE
Total available budget always shows 0.00$ + No progress bars

### DISCUSSION THREAD

**Discussion by ghost** [firefly-iii/firefly-iii#8962] [Firefly III] [answered]
Hello Every one,

![image](https://github.com/firefly-iii/firefly-iii/assets/57751312/704707f1-82a9-40d1-b745-87ae97741c8c)

Can some one help me with this issue ? The Available amount is always 0.00 with no progress bars.

I tried to change the budget but dosent work. I'm using SQLite database

Thanks

**Comment by JC5** [comment, marked as answer]
There is a related bug that could be the fix for this in the next release, but I can't be sure. I'll check it out this week, see if I can replicate it.
